queries.json

```
@@ -47,19 +47,34 @@
       "description": "Enumerates the classful networks for all destination IP addresses including count of connections"
     },
     {
-      "name": "Suricata Alerts by Category",
+      "name": "Suricata Alerts by Severity & Category",
```

Nit: Other queries use "and" instead of "&".
```suggestion
      "name": "Suricata Alerts by Severity and Category",
```